### PR TITLE
Create UserEndpoint

### PR DESCRIPTION
**Pull request by BrunoMilitoFrancois**

### CODE REVIEW

File: ZPastel.Test/UserEndpointTests.cs [hunk 1]
```diff
@@ -0,0 +1,180 @@
...
+            var content = new StringContent(JsonSerializer.Serialize(user), Encoding.UTF8, "application/json");
+
+            var createResponse = await client.PostAsync("api/users/save", content);
+            createResponse.StatusCode.Should().Be(HttpStatusCode.OK);
+
+            var createdUserContent = await createResponse.Content.ReadAsStringAsync();
+            var userResource = Newtonsoft.Json.JsonConvert.DeserializeObject<UserResource>(createdUserContent);
+
+            userResource.Should().NotBeNull();
+            userResource.CreatedById.Should().Be(1);
```
Comment (joseluz) [inline ZPastel.Test/UserEndpointTests.cs]: Use the updated object "user"

File: ZPastel.Test/UserEndpointTests.cs [hunk 2]
```diff
@@ -0,0 +1,180 @@
...
+            userResource.FirebaseId.Should().Be("user-1");
+            userResource.Id.Should().Be(1);
+            userResource.LastModifiedById.Should().Be(2);
+            userResource.LastModifiedOn.Should().Be(now);
+            userResource.Name.Should().Be("New Name");
+            userResource.PhotoUrl.Should().Be("www.newurl.com/photo.jpg");
+        }
+
+        [Fact]
+        public async Task SaveUser_WithEmptyName_ShouldThrowArgumentException()
```
Comment (joseluz) [inline ZPastel.Test/UserEndpointTests.cs]: Update the test name to ...ShouldReturnBadRequest

Comment (joseluz) [reply ZPastel.Test/UserEndpointTests.cs]: Check in this test if the operation return which field is causing the problem

File: ZPastel.Test/UserEndpointTests.cs (hunk 1 above)
Comment (BrunoMilitoFrancois) [reply ZPastel.Test/UserEndpointTests.cs]: Done.

File: ZPastel.Test/UserEndpointTests.cs (hunk 2 above)
Comment (BrunoMilitoFrancois) [reply ZPastel.Test/UserEndpointTests.cs]: Done.

**Pull request merged**